src/Sulu/Bundle/ContactBundle/Controller/AbstractMediaController.php

```diff
@@ -120,4 +134,333 @@ protected function removeMediaFromEntity($entityName, $id, $mediaId)
 
         return $this->handleView($view);
     }
+
+    /**
+     * Returns a view containing all media of an entity.
+     *
+     * @param String $entityName
+     * @param String $routeName
+     * @param AbstractContactManager $entityManager
+     * @param String $id
+     * @param Boolean $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    protected function getMultipleView($entityName, $routeName, AbstractContactManager $entityManager, $id, $request)
```

Comment:
Rename `$entityManager` to `$contactManager` or `$abstractContactManager`. An entity manager is something different :wink: